### ISSUE TITLE
Mouse lag when move over menus on Mac client

### ISSUE THREAD

**Issue by CthulhuCalamari** [symless/synergy-core#6172]
I can appreciate that there might be some "wake up" time when the mouse has sat inactive on the client window for a period of time, but it is less understandable when it happens while in use on menus. Whenever the mouse interacts with a menu on the client, it lags. This happens virtually every time, and is especially problematic on the Mac OS Dock. For drop-down/drop-down menus that disappear when the mouse leaves the window, it is even more frustrating.

### Operating Systems ###

Server: Win10, Version 1703 (OS Build 15063.674)
Client: Mac OS X 10.11.6

### Synergy Version ###

1.8.8, March 16, 2017 (verified on both comps)

### Steps to reproduce bug ###

1. Mouse over/select any menu that drops down/pops out/expands/etc. on the client screen
2. Be frustrated that the mouse lags when navigating the resulting menu

### Other info ###

* I do not recall this being an issue when I installed the program on Tuesday. It became frustrating on Wednesday. It remained frustrating enough to post on a forum on Thursday. (Win10 on the server was updated last night, but the problem occurred before _and_ after this.)
* The only current work-around is using the Mac mouse to operate on the client screen, which negates the whole point of using Synergy
* This doesn't stop me from using Synergy entirely, but it _does_ discourage me from recommending it to anyone.

Debug Logs
Server: Nope
Client: Not a chance

The process of trying to create debug logs has also been a colossal pain. Both the Server & Client sides spit out so much data that the logs individually go over the free Pastebin limit, and I'm not paying to explain the problem with the program I paid for. When I tried to copy the Client log I found that the Server mouse still had to be used, despite closing the Server-side program in the Task Manager, which is probably a _whole_ other problem... Restarting both and attempting to get a smaller log from each once they had just started up was not only fruitless because they'd already created too much data, but also because it caused a host of other problems, not least of all the mouse disappearing on _both_ computers. Out of curiosity, I pasted the Server log into Word to see just how much data there was, and it spit out 1692 pages. One _THOUSAND_, six hundred and ninety-two pages.

This is extremely untenable.

**Comment by CthulhuCalamari** [comment]
UGH.

Version 1.9 is posted; we'll see if that fixes this:
https://symless.com/forums/topic/2587-download-synergy-19-rc3-fixes-the-macos-wifi-lag-bug/

**Comment by CthulhuCalamari** [comment]
UGH.

Version 1.9 is posted; we'll see if that fixes this:
https://symless.com/forums/topic/2587-download-synergy-19-rc3-fixes-the-macos-wifi-lag-bug/

**Comment by CthulhuCalamari** [comment]
NOPE. Still lags. It's an improvement, but not a fix.

**Comment by melMass** [comment]
How are your computer's connected together ? Wifi, Ethernet ?

**Comment by CthulhuCalamari** [comment]
> How are your computer's connected together ? Wifi, Ethernet ?

They're over wifi, but it's on my office network, which is really solid.

**Comment by duchy** [comment]
Seem issue on my side. Even with version 2.0.0-stable.

**Comment by svaningelgem** [comment]
Same issue here.
Server is Win10, Client is Mac OS 10.13.

The issue is a general lag. Mouse moves and key entries are with a delay.
Before I was working on Mac OS 10.10, and there I didn't have any issues (but I had to upgrade because of a necessary program that needed at least 10.12).

(according to the internet, it seems to have been solved in 1.9rc3, but I still have it in the 1.10.3 latest release.)
